packages/ember-data/lib/system/relationships/ext.js

```diff
@@ -273,13 +272,13 @@ Model.reopenClass({
       inverseKind = possibleRelationships[0].kind;
     }
 
-    function findPossibleInverses(type, inverseType, relationshipsSoFar) {
+    function findPossibleInverses(modelClass, inverseType, relationshipsSoFar) {
```

Comment:
the factory was called `typeClass` earlier, and here is `modelClass`. Should probably pick one.